docs/readme.md

```diff
@@ -0,0 +1,29 @@
+---
+id: readme
+title: Welcome to FAST-DNA
+sidebar_label: What is FAST-DNA
+---
+
+Check the [documentation](https://docusaurus.io) for how to use Docusaurus.
+
+## Lorem
```

Comment:
Do we need lorem ipsum? it just adds noise and makes it seem more complete than it is - providing less incentive to actually put in documentation.

I'd recommend removing.